packages/prettier-plugin-java/src/printers/blocks-and-statements.js

```diff
@@ -56,7 +59,7 @@ class BlocksAndStatementPrettierVisitor {
       return this.visitSingle(ctx);
     }
 
-    return this.getSingle(ctx).image;
+    return getImageWithComments(this.getSingle(ctx));
```

Comment:
I am not quite sure about that, since the purpose of this function is to just retrieve the single node existing node from a parent node. There is one case where we do not only want to get the token and we want to traverse the [nodes](https://github.com/jhipster/prettier-java/blob/master/packages/prettier-plugin-java/src/cst-printer.js#L61-L64)